docs/release_docs/Uat-ci-cd_ReleaseFLow.md

```diff
@@ -0,0 +1,33 @@
+## UAT-CI-CD
+
+There are two ways to use or trigger uat-CI-CD workflow
```

Comment:
Could you add a section at the top of every workflow explanation, stating the mode of trigger for the workflow? 
The values for it will be either `On-Demand` or `Automated`

So in the case of UAT, it will be both `On-Demand` and `Automated` and the PR, DEV and Prod will be automated.